security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -315,13 +320,13 @@ func (sf *SecretFetcher) scrtUpdated(oldObj, newObj interface{}) {
 		return
 	}
 
-	oldCert, oldKey, oldRoot, _ := extractCertAndKey(oscrt)
-	newCert, newKey, newRoot, valid := extractCertAndKey(nscrt)
+	oldCert, oldKey, oldRoot, oldCrl, _ := extractCertAndKey(oscrt)
```

Comment:
i rarely see functions returned 5 returned values... it's hard to remember the order and the meaning of each, we should define struct to wrap.